raiden-dapp/CHANGELOG.md

```diff
@@ -2,7 +2,7 @@
 
 ## [Unreleased]
 ### Fixed
-- [#2026] Removed delay time on tooltips
+- [#2237] Removed delay time on tooltips
```

Comment:
Wut?

Reply:
Yes, something went wrong. I'm fixing it. :)